tests/controller/accountscontrollertest.php

```diff
@@ -203,17 +204,17 @@ public function testCreateAutoDetectFailure() {
 
 	public function newMessageDataProvider() {
 		return [
-			[false, false],
-			[true, false],
-			[false, true],
-			[true, true],
+			[false, false, false],
+			[true, false, false],
+			[false, true, false],
+			[true, true, true],
```

Comment:
looks good, just one question, do we need to check for other cases like [false, false, true] [true, true, false] etc.